setup.cfg

```diff
@@ -53,6 +53,103 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
```

Comment:
Could you add `pytest-asyncio` here? https://github.com/pandas-dev/pandas/pull/48361

Reply:
Done.  Went for latest version as pin. pytest-asyncio>=0.19.0